src/clusterfuzz/_internal/bot/tasks/cron/sync_admins.py

```diff
@@ -79,5 +79,10 @@ def main():
       resource=project_id, body={}).execute()
 
   admins = admins_from_iam_policy(policy)
-  update_admins(admins)
-  return True
\ No newline at end of file
+  try:
+    update_admins(admins)
+    logs.log('Sync admins succeeded.')
+    return True
+  except Exception as e:
```

Comment:
nit: it's better to just letter the exception bubble up in these cases. Any reason to prefer catching it here?